wgsl/index.bs

```diff
@@ -520,6 +520,76 @@ type.
 
 ## Texture Types TODO ## {#texture-types}
 
+A <dfn noexport>texel</dfn> is a scalar or vector used as the smallest independently addressible element of a [=texture=].
+The word *texel* is short for *texture element*.
+
+A <dfn noexport>texture</dfn> is a grid of texels supporting special operations useful for rendering.
+A texture's representation is typically optimized for those operations, with many details hidden from the programmer
+including data layouts, data types, and operations that cannot be expressed directly in the shader language.
+Each texel has a unique grid coordinate within the texture.
+
+An <dfn noexport>arrayed texture</dfn> is a texture which is organized as a homegeneous array of grids of texels.
```

Comment:
nit: isn't the arrayness just an extra dimension for the multi-dimensional grid?

Reply:
I was hoping to treat the array index specially. The regular grid coordinates have interpolation applied to them, but the array index does not.  I suggest we add a note to revisit this editorially